setup/security/admin/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: single
+title:  "Administrator functionality"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+## Introduction
+In Spinnaker, it is possible to define that users belonging to a certain role are considered "Administrators". This virtually removes all READ/WRITE restrictions to accounts and applications for these users.
+
+{% include
+   warning
+   content="This feature gives God Mode like capabilities to the users who are admins. Proceed with caution."
+%}
+
+## Enable and Configure Admin functionality
+
+### Halyard
+TBD
+
+### Manually add configuration in fiat
+
+In the fiat config file, add the following:
+
+```yaml
+fiat:
+  admin:
+    roles:
+      - "role of the spinnaker gods"
```

Comment:
It might be more useful to have an example of what should actually go here. Something like:
```
- devops-admin
```

Reply:
maybe a note on if this using saml service (okta) then these are okta groups and you would need to run `hal deploy apply` after the change.